flixel/system/frontEnds/BitmapFrontEnd.hx

```diff
@@ -114,7 +114,7 @@ class BitmapFrontEnd
 	/**
 	 * Check the local bitmap cache to see if a bitmap with this key has been loaded already.
 	 *
-	 * @param	Key		The string key identifying the bitmap.
+	 * @param	Key	The key identifying the bitmap.
```

Comment:
- make `key` lower case
- put 2 spaces after `key` and 3 before (currently a tab)